map/src/map/layers/LocalClientTrackLayer.js

```diff
@@ -186,7 +189,11 @@ export default function LocalClientTrackLayer() {
 
     function saveLocal() {
         if (ctx.localTracks.length > 0) {
+            // localTracks exist: do update/append
```

Comment:
delete comment

Reply:
Comment describes ambiguous code: we must understand, why we calling createLocalTrack() in some case and saveTracks() in another.

There was a strange code which lead to the hidden bug, so this comments are really useful to remember the reasons of bug and also to undertand that saveTracks() don't work correctly if there is no tracks was saved before.

Finally, my opinion is to keep this kind of comments as useful for me and other developers.